traffic_ops/testing/api/v4/cdnnotifications_test.go

```diff
@@ -0,0 +1,61 @@
+package v4
+
+/*
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+import (
+	"github.com/apache/trafficcontrol/lib/go-tc"
+	"testing"
```

Comment:
standard library imports should be grouped before project-local imports

Reply:
i guess my "optimize imports" button doesn't work :(

Reply:
That's a weird-sounding button